tests/src/functional/Drush/Traits/SiteUriTraitTest.php

```diff
@@ -51,6 +51,40 @@ public function testGetSitesSubdirFromUri(): void {
     $uri = $this->getSitesSubdirFromUri($drupal_root, "http://www.acquia.com");
     // If URI does't exist on sites.php file, fallback to site default.
     $this->assertEquals($uri, "default");
+
+    // If uri makes pattern port.domain.path and it exists in the sites.php
+    // then it return the value.
+    $uri = $this->getSitesSubdirFromUri($drupal_root, "http://acquia.com/stage/acquia_cms");
+    $this->assertEquals($uri, "stage.acquia_cms");
+
+    $uri = $this->getSitesSubdirFromUri($drupal_root, "http://acquia.org:8080/developer/acquia_cms");
+    $this->assertEquals($uri, "developer.acquia_cms");
```

Comment:
Same here, This should be acquia.org instead developer.acquia_cms, right?

Reply:
Same se above.